### PR TITLE
gnu-efi/shim for arm64

### PR DESCRIPTION
**Pull request by mischief**
still needs to be moved to the board, but this at least allows shim to compile correctly.

depends on coreos/portage-stable#410

### CODE REVIEW

**Comment by mischief** [comment]
retest this please

**Comment by mischief** [comment]
retest this please

**Comment by marineam** [comment]
Need to update KEYWORDS

**Comment by mischief** [comment]
done

File: sys-boot/shim/shim-9999.ebuild [hunk 1]
```diff
@@ -6,13 +6,13 @@ CROS_WORKON_PROJECT="coreos/shim"
 CROS_WORKON_REPO="git://github.com"
 
 if [[ "${PV}" == 9999 ]]; then
-	KEYWORDS="~amd64"
+	KEYWORDS="~amd64 arm64"
```
Comment (marineam) [inline sys-boot/shim/shim-9999.ebuild]: ~arm64

**Comment by mischief** [comment]
ptal

**Comment by marineam** [comment]
LGTM

**Pull request merged**